pinc/Activity.inc

```diff
@@ -145,7 +145,7 @@ class Activity
         $this->access_change_callback = $access_change_callback;
 
         global $testing;
-        if ($testing && !is_null($this->access_minima)) {
```

Comment:
Feels like we shouldn't remove this check until we update the Activity constructor with type declarations so that we can guarantee `access_minima` is an array. Because while it happens to be with today's code, it doesn't mean that it will always be and I don't want to rely on phpstan alone.

Reply:
OK